README.md

```diff
@@ -62,10 +62,13 @@ The nettools Lego could be triggered by various prefix.
 
 ` !ping {target}`
 
-#### resolver
+#### Resolver
```

Comment:
Is this the same as nslookup?

Reply:
The resolver implements nslookup :) i just added the host command to the package so its name changed.

Reply:
Can we alias `!nslookup` to the resolver class as well, then? I feel like people would try to use the tool by the name they are more familiar with.

Reply:
Yep, idk when I will have time to patch it but I will do it ASAP